src/post/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+class Post extends React.Component {
+
+    render() {
+        const { post } = this.props;
+        const { photoUrl, caption } = post;
+
+        return (
+            <>
+                <Photo photoUrl={photoUrl} />
```

Comment:
you forgot to import photo and caption component

Reply:
🤔